plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/TemplateParametersPage.java

```diff
@@ -124,7 +124,13 @@ protected void doCreateControls(Composite container, DataBindingContext dbc) {
 		resetButton.setText("Reset");
 		UIUtils.setDefaultButtonWidth(resetButton);
 		resetButton.addSelectionListener(onReset());
-
+		ValueBindingBuilder
+		.bind(WidgetProperties.enabled().observe(resetButton))
+		.notUpdatingParticipant()
+		.to(selectedParameter)
+		.converting(new IsNotNull2BooleanConverter())
+		.in(dbc);
+		
```

Comment:
First you make him do something he doesn't want to... and then you rub it in his face and tell everyone.  Have you no common courtesy, Andre?  What about poor Fred's feelings.